sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsyncBenchmark.java

```diff
@@ -90,13 +91,17 @@
             .clientTelemetryEnabled(cfg.isClientTelemetryEnabled());
 
         if (registry != null) {
-            cosmosClientBuilder.clientTelemetryConfig().clientMetrics(registry);
+            CosmosClientTelemetryConfig telemetryConfig = new CosmosClientTelemetryConfig();
+            telemetryConfig.clientMetrics(registry);
```

Comment:
what do you think about

```suggestion
            telemetryConfig.setMeterRegistry(registry);
```

?

Reply:
Fixed